rr_arduino/rr_arduino.ino

```diff
@@ -0,0 +1,212 @@
+/*
+ * @file Robot Racer Controller                                                       
+ * @author Tom Meredith
+ * @author Noah Abradjian 
+ * @author Toni Ogunmade  
+ * @author Abhi Srikantharajah 
+ * @author Brian Kibazohi
+ * @competition IARRC 2018
+ * Last Updated: June 1, 2018                                                   
+ */
+
+//#define TEST_OUTPUT
+
+//Import Libraries
+#include <SPI.h>
+#include <Servo.h>
+#include <robot_racer.h>
+//#define USE_USBCON
+#include <ros.h>
+#include <std_msgs/Int8.h>
+#include <std_msgs/Float32.h>
+#include <std_msgs/Float32MultiArray.h>
+#include <PID_v1.h>
+
+
+//brief Serial, velocity and battery monitoring defines respectively
+#define ROS_BAUD_RATE         57600
+#define EC_BAUD_RATE          115200
```

Comment:
change the defines to [const](https://google.github.io/styleguide/cppguide.html#Constant_Names) ints(or whatever datatype matches) 
This is because of [this](https://google.github.io/styleguide/cppguide.html#Preprocessor_Macros)